src/OrbitGl/CallTreeView.cpp

```diff
@@ -87,17 +87,6 @@ CallTreeUnwindErrors* CallTreeNode::AddAndGetUnwindErrors() {
   return unwind_errors_child_.get();
 }
 
-uint64_t CallTreeNode::GetExclusiveSampleCount() const {
-  uint64_t children_sample_count = 0;
-  for (const auto& address_and_function : function_children_) {
-    children_sample_count += address_and_function.second.sample_count();
-  }
-  for (const auto& tid_and_thread : thread_children_) {
-    children_sample_count += tid_and_thread.second.sample_count();
-  }
```

Comment:
Note that there was actually a little bug here because we were not adding to `children_sample_count` from `unwind_errors_child_`, but that had no effect because every node with a `unwind_errors_child_` doesn't show its exclusive percentage/count.